edge-rtac/src/main/resources/domain/edge-rtac/features/rtac.feature

```diff
@@ -0,0 +1,153 @@
+Feature: rtac tests
+
```

Comment:
It looks like all these tests are creating and attaching items.  Yet there is behavior specified in edge-rtac for returning holdings records if there are no attached item records (I know cos I put it in :).  Have I missed the test for this?  If not, one needs to be added.

Reply:
@K-Felk  
Yes , when we send param fullperiodicals=false then api only returns holdings data for periodical data , no matter if item is attached to the holdings or not.

EDGRTAC-48 :- **Create Karate test for Edge-RTAC: For periodical/serial, return only holdings information including availability for each instance UUID included in request WHEN &fullPeriodicals=false OR no parameter is omitted (edge-rtac-4)**

Is this you are talking about or i am mistaking ?

Reply:
You're right, I missed it.  Mea culpa.